test/commands/test/generate/uitest-test.ts

```diff
@@ -66,31 +65,25 @@ describe("Validating UITest template generation", () => {
       });
     });
 
-    const packageFilePath = path.join(command.outputPath, `AppCenter.UITest.${command.platform}/packages.config`);
     const projectFilePath = path.join(
       command.outputPath,
       `AppCenter.UITest.${command.platform}/AppCenter.UITest.${command.platform}.csproj`
     );
 
     // Assert
-    let packageFileContent = await pfs.readFile(packageFilePath, "utf8");
-    expect(packageFileContent).not.contain(fakeLatestVersion);
-
     let projectFileContent = await pfs.readFile(projectFilePath, "utf8");
     expect(projectFileContent).not.contain(fakeLatestVersion);
 
     // Act
     await (command as any).processTemplate();
 
     // Assert
-    packageFileContent = await pfs.readFile(packageFilePath, "utf8");
-    expect(packageFileContent).contain(fakeLatestVersion);
-
     projectFileContent = await pfs.readFile(projectFilePath, "utf8");
     expect(projectFileContent).contain(fakeLatestVersion);
   });
 
   it("should not touch NuGet version on failure", async () => {
+    const latestVersion = "3.13.3";
```

Comment:
where this number is coming from?

Reply:
It is version set in template. See https://github.com/microsoft/appcenter-cli/pull/2179/files#diff-43a9902987546c39cfb52668af857cb9dcda87d4b0e4a3aef24f9b102c8ed6d0R29

The test makes sure the version won't be changed after executing command with a failure.

Reply:
Actually, I think we need to test X.UITest version instead of version of NUnit.

Reply:
Done